example/tutorial/Hierarchical_Parallelism/04_team_scan/team_scan.cpp

```diff
@@ -48,11 +48,11 @@
 #include <cstdio>
 #include <cstdlib>
 
-typedef Kokkos::DefaultExecutionSpace Device;
-typedef Kokkos::HostSpace::execution_space Host;
+using Device = Kokkos::DefaultExecutionSpace;
+using Host   = Kokkos::HostSpace::execution_space;
 
-typedef Kokkos::TeamPolicy<Device> team_policy;
-typedef team_policy::member_type team_member;
+using team_policy = Kokkos::TeamPolicy<Device>;
+using team_member = team_policy::member_type;
```

Comment:
I'm kind of surprised this doesn't require `typename`?

Reply:
because it is not a template